community/kernel/src/main/java/org/neo4j/storageengine/api/StorageStatement.java

```diff
@@ -68,19 +69,21 @@
      * to place the cursor over the first item and then more calls to move the cursor through the selection.
      *
      * @param nodeId id of node to get cursor for.
+     * @param assertOpen to check if source transaction is still open on each property value fetch
```

Comment:
i think anyone are free to leave comment if they think it can make PR better in one way or another